core/runtime/src/main/java/io/quarkiverse/langchain4j/audit/AuditSourceInfoImpl.java

```diff
@@ -0,0 +1,82 @@
+package io.quarkiverse.langchain4j.audit;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import io.quarkiverse.langchain4j.runtime.aiservice.AiServiceMethodCreateInfo;
+
+/**
+ * Contains information about the source of an audit event
+ */
+public final class AuditSourceInfoImpl implements AuditSourceInfo {
```

Comment:
Does this have to be public?

Reply:
I supposed I could move the implementation class elsewhere (to `io.quarkiverse.langchain4j.runtime.aiservice`)

Reply:
Yeah, I would much prefer that

Reply:
I don't want the class to be visible

Reply:
Let me know what you'd like me to do. We either leave it alone, or we move the class to the `io.quarkiverse.langchain4j.runtime.aiservice` package and remove `public` from it.

My $0.02 I think we leave it as is. I think it makes more sense to keep it packaged with the rest of the auditing stuff rather than moving some random auditing class into a package dealing with AiService stuff. But I'll do whatever you prefer.

Reply:
Let's move it, I don't want it to be public

Reply:
weird it took forever for your comments to show up for me